ProcessLib/HT/CreateHTProcess.cpp

```diff
@@ -92,12 +70,18 @@ std::unique_ptr<Process> createHTProcess(
         "density_solid", parameters, 1);
     DBUG("Use \'%s\' as density_solid parameter.", density_solid.name.c_str());
 
+    //! \ogs_file_param_special{prj__processes__process__HT__fluid__density}
+    auto const& fluid_density_conf = fluid_config.getConfigSubtree("density");
+    auto fluid_density =
+        MaterialLib::Fluid::createFluidDensityModel(fluid_density_conf);
+
     // Parameter for the density of the fluid.
-    auto& density_fluid = findParameter<double>(
+    auto& fluid_reference_density= findParameter<double>(
         config,
-        //! \ogs_file_param_special{prj__processes__process__HT__density_fluid}
-        "density_fluid", parameters, 1);
-    DBUG("Use \'%s\' as density_fluid parameter.", density_fluid.name.c_str());
+        //! \ogs_file_param_special{prj__processes__process__HT__fluid_reference_density}
+        "fluid_reference_density", parameters, 1);
```

Comment:
Not sure, but maybe it should be `prj__processes__process__HT__fluid__reference_density` (extra underscore after fluid)?